www/front_src/src/Resources/Graph/Performance/AnomalyDetection/ThresholdAD.tsx

```diff
@@ -0,0 +1,98 @@
+import { equals, prop, isNil } from 'ramda';
+import { Threshold } from '@visx/threshold';
+import { curveBasis } from '@visx/curve';
+import { ScaleLinear, ScaleTime } from 'd3-scale';
+
+import { TimeValue } from '../models';
+import { getTime } from '../timeSeries';
+
+interface Props {
+  getYScale;
+  graphHeight: number;
+  leftScale: ScaleLinear<number, number>;
+  regularLines;
+  rightScale: ScaleLinear<number, number>;
+  secondUnit: string;
+  thirdUnit: string;
+  timeSeries: Array<TimeValue>;
+  xScale: ScaleTime<number, number>;
+}
+
+const ThresholdAD = ({
```

Comment:
```suggestion
const AnomalyDetectionEnvelopeThreshold = ({
```

Does it make any sense to add some memoization here? cc @Thebarda

Reply:
As it is a heavy component, it makes sense to add memoization on non-function props